project/Game.py

```diff
@@ -170,6 +174,24 @@ def modify_landmark(self, oldclue, clue=None, question=None, answer=None):
         except KeyError:
             return False
 
+    def edit_landmark_order(self, oldindex, newindex):
+        if not self.started:
+            try:
+                index1 = int(oldindex)
+                index2 = int(newindex)
+            except ValueError:
+                return Errors.STRING_TO_INTEGER
+            if index1 < 0 or index2 < 0:
+                return Errors.NEGATIVE_INDEX
+            try:
+                temp = self.__landmarks[index2]
+                self.__landmarks[index2] = self.__landmarks[index1]
+                self.__landmarks[index1] = temp
```

Comment:
This code seems to be doing a swap positions. So it's changing the "relative" order of two landmarks, not just one. My thought was that we would do an insert at the new index and a remove at the old index, only changing the relative location of one landmark.

Reply:
wait what do you mean? not quite following

Reply:
oh i get what you mean, but than the command should not be "editlandmarkorder", not quite clear to what you described.

Reply:
sounds like you are describing doing in simple terms -> an add command followed by a remove? is that right?